src/compiler/transformers/rewrite_aliased_paths.ts

```diff
@@ -0,0 +1,152 @@
+import { normalizePath } from '@utils';
```

Comment:
Not about this line, just a general comment about the file. Can we update the name of this file to be kebab-case instead of snake-case (i.e. `rewrite-aliased-paths.ts`)